custom_components/solcast_solar/manifest.json

```diff
@@ -10,5 +10,5 @@
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/BJReplay/ha-solcast-solar/issues",
   "requirements": ["aiohttp>=3.8.5", "aiofiles>=23.2.0", "datetime>=4.3", "isodate>=0.6.1"],
-  "version": "4.2.3"
+  "version": "4.2.3.1"
```

Comment:
Just checking that this shouldn't be another number - putting this in so that stick but I assume it will get fixed by the time we get to a release.

Reply:
Yeah. The version there is for publishing an alpha at autoSteve/ha-solcast-solar. I probably should kill this PR and start again, but left it so you could follow along until it's baked.

GitHub won't allow a merge with manifest as it is because conflict. A handy reminder.

Reply:
Crumbs - just reading my earlier comments. Ugh. Comfortable Screen <-> Eye distance != Comfortable Keyboard <-> Chair distance. Leading to gobbledygook comments.

But you got the point.